### PR TITLE
Summarize all breaking changes in diffs at the top of the migration guide

### PR DESCRIPTION
**Pull request by sujuka99**

### CODE REVIEW

File: docs/docs/user/migration-guide/v2-v3.md [hunk 1]
```diff
@@ -236,3 +238,100 @@ This would make it more uniform with the existing `${component.<key>}` variables
...
+      ...
+      - type: kafka-app
+        app:
+    -     brokers: ${brokers}
+    +     brokers: ${config.kafka_brokers}
+          labels:
+    -       app_schedule: "${component_app_schedule}"
+    +       app_schedule: "${component.app.schedule}"
+      ...
+      - type: kafka-connector:
```
Comment (disrupted) [inline docs/docs/user/migration-guide/v2-v3.md:274]:
```suggestion
      - type: kafka-connector
```

**Review by disrupted — CHANGES_REQUESTED** [review]

File: docs/docs/user/migration-guide/v2-v3.md [hunk 2]
```diff
@@ -236,3 +238,100 @@ This would make it more uniform with the existing `${component.<key>}` variables
 +     brokers: ${config.kafka_brokers}
 +     schemaRegistryUrl: ${config.schema_registry.url}
 ```
+
+## Summary
+
+!!! warning
```
Comment (raminqaf) [inline docs/docs/user/migration-guide/v2-v3.md:244]: didn't we ignore these with 🤔 
```yaml
<!-- dprint-ignore-start -->
<!-- dprint-ignore-end -->
```

File: docs/docs/user/migration-guide/v2-v3.md [hunk 3]
```diff
@@ -236,3 +238,100 @@ This would make it more uniform with the existing `${component.<key>}` variables
...
+
+    [**Helm will not find your (long) old release names anymore.**](#use-hash-and-trim-long-helm-release-names-instead-of-only-trimming)
+
+??? example "defaults.yaml"
+
+    ```diff
```
Comment (raminqaf) [inline docs/docs/user/migration-guide/v2-v3.md:250]: Just double-check if it is correctly rendered on the `dev` branch of the docs.
the .md file looks a bit strange
image

Comment (raminqaf) [reply docs/docs/user/migration-guide/v2-v3.md:250]: nvm looks great
image

File: docs/docs/user/migration-guide/v2-v3.md (hunk 2 above)
Comment (sujuka99) [reply docs/docs/user/migration-guide/v2-v3.md:246]: Here it wasn't a problem apparently, but I did add the comments just in case

**Review by disrupted — APPROVED** [review]

**Pull request merged**